appcli/cli_builder.py

```diff
@@ -224,106 +213,9 @@ def check_docker_socket():
         """Check that the docker socket exists, and exit if it does not
         """
         if not os.path.exists("/var/run/docker.sock"):
-            error_msg = f"""Please relaunch using:
-
-    docker run \\
-        --rm
-        --volume /var/run/docker.sock:/var/run/docker.sock \\
-        {configuration.docker_image}:{APP_VERSION} \\
-            --configuration-dir <dir> \\
-            --data-dir <dir> COMMAND'
-            --environment <str>
-"""
+            error_msg = f"""Docker socket not present. Please launch with a mounted `docker.sock`."""
```

Comment:
Not as explicit as I would like. Please use:
"""Docker socket not present. Please launch with a mounted `/var/run/docker.sock`"""